src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -3,67 +3,83 @@ package org.malachite.estella.task.api
 ) {
 
     @CrossOrigin
     @Transactional
     @GetMapping
-    fun getTaskByRecruitmentProcess(@RequestParam(name = "process") processId: Int): ResponseEntity<List<TaskDto>> =
-        taskService
-            .getTasksByRecruitmentProcess(processId)
+    fun getTaskByRecruitmentProcess(
+        @RequestParam("owner") organizationUuid: String,
+        @RequestHeader(EStellaHeaders.passwordHeader) password: String
+    ): ResponseEntity<List<TaskDto>> =
+        taskService.checkDevPassword(organizationUuid, password)
+            .getTasksByOrganizationUuid(organizationUuid)
             .let { ResponseEntity.ok(it) }
 
     @Deprecated(message = "Wasn't tested yet - unnecessary now - to be implemented and tested in ES-17 epic")
     @CrossOrigin
     @Transactional
-    @GetMapping("/{taskId}")
-    fun getTaskById(@PathVariable taskId: Int) = ResponseEntity(Message("Not Implemented"), HttpStatus.NOT_IMPLEMENTED) /*taskService.getTaskById(taskId)
+    @GetMapping("{taskId}")
+    fun getTaskById(
+        @RequestParam("owner") organizationUuid: String,
+        @PathVariable taskId: Int,
+        @RequestHeader(EStellaHeaders.passwordHeader) password: String
+    ) =
+        ResponseEntity(Message("Not Implemented"), HttpStatus.NOT_IMPLEMENTED) /*taskService.getTaskById(taskId)
         .let { ResponseEntity.ok(it) }*/
 
     @Deprecated(message = "Wasn't tested yet - unnecessary now - to be implemented and tested in ES-17 epic")
     @CrossOrigin
     @Transactional
-    @GetMapping("/{taskId}/tests")
-    fun getTaskTests(@PathVariable taskId: Int) = ResponseEntity(Message("Not Implemented"), HttpStatus.NOT_IMPLEMENTED)
+    @GetMapping("{taskId}/tests")
+    fun getTaskTests(
+        @RequestParam("owner") organizationUuid: String,
+        @PathVariable taskId: Int,
+        @RequestHeader(EStellaHeaders.passwordHeader) password: String
+    ) =
+        ResponseEntity(Message("Not Implemented"), HttpStatus.NOT_IMPLEMENTED)
     /*taskService.getTestsFromTask(taskId)
         .let { ResponseEntity.ok(it) } */
 
     @CrossOrigin
     @Transactional
-    @PostMapping("/{processId}")
+    @PostMapping
     fun addTask(
-        @PathVariable processId: Int,
+        @RequestParam("owner") organizationUuid: String,
+        @RequestHeader(EStellaHeaders.passwordHeader) password: String,
         @RequestBody task: TaskDto
-    ) = taskService.addTask(task).let { t ->
-        processService.getProcess(processId)
-            .let {
-                processService.updateTasks(processId, setOf(t.toTask()))
-            }
-    }
+    ) = taskService.checkDevPassword(organizationUuid, password).addTask(task)
+        .let { t -> organizationService.updateTasks(organizationUuid, setOf(t.toTask())) }
         .let { OwnResponses.SUCCESS }
 
     @Deprecated(message = "Wasn't tested yet - unnecessary now - to be implemented and tested in ES-17 epic")
     @CrossOrigin
     @Transactional
     @PutMapping("/{taskId}/tests/file")
```

Comment:
Nie wiem, czy te dwa endponinty (ten i chyba następny) nie są do wyrzucenia

Reply:
Pytanie czy chcemy mieć możliwość edycji samych testów.

Reply:
No jakby test który wrzucili był niepoprawny to fajnie jakby go można było naprawić

Reply:
Wait what? XDDD

Reply:
No dev dorzuca testy ale po jakiejś rozmowie okazuje się że są one zepsute bo czegoś nie uwzględnił jakiegoś edge case'u cokolwiek, no to fajnie jakby można było je zmienić